atlp-rwanda/ecommerce-app-predators-fe
# Hotfix: Test coverage configuration

#### What does this PR do?
- Configures jest to collect coverage from `src/` directory

#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout hotfix-test-coverage
npm i
npm run test
```
You should see a coverage report of all relevant files in `src/` directory.

#### What are the relevant pivotal tracker/Trello stories?
[Card #47](https://trello.com/c/DuD2CSVg)
